### PR TITLE
Feat/modal animation

### PR DESCRIPTION
**Pull request by ytakayanagi**
Closes # 2519

## 📝 Description

Added option "none" for motionPreset prop in modal component to give an option to disable animation.

## ⛳️ Current behavior (updates)

Currently there is no option to turn off the animation in modal component.
## 🚀 New behavior

User will now have an option to specify "none" in motionPreset prop to disable the modal animation.

## 💣 Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: packages/modal/src/modal-transition.tsx [hunk 1]
```diff
@@ -30,6 +31,7 @@ export const ModalTransition = React.forwardRef(
   (props: ModalTransitionProps, ref: React.Ref<any>) => {
     const { preset, ...rest } = props
     const motionProps = transitions[preset]
+    console.log(motionProps)
```
Comment (TimKolberger) [inline packages/modal/src/modal-transition.tsx:34]: Please remove this logging statement

**Review by TimKolberger — CHANGES_REQUESTED** [review]
Thank your for this PR @ytakayanagi! 

I would love to see the new motion preset mentioned in the docs at [Modal#changing-the-transition](https://chakra-ui.com/docs/overlay/modal#changing-the-transition).

File: packages/modal/src/modal-transition.tsx (hunk 1 above)
Comment (ytakayanagi) [reply packages/modal/src/modal-transition.tsx:34]: Thank you for the review! I will  remove the logging statement and add the new motion preset information in the docs.

File: website/pages/docs/overlay/modal.mdx [hunk 2]
```diff
@@ -299,7 +299,7 @@ function VerticallyCenter() {
 
 The `Modal` comes with a scale transition by default but you can change it by
 passing a `motionPreset` prop, and set its value to either `slideInBottom`,
-`slideInRight`, or `scale`.
+`slideInRight`, or `scale`, or `none`.
```
Comment (TimKolberger) [inline website/pages/docs/overlay/modal.mdx:302]:
```suggestion
`slideInRight`, `scale` or `none`.
```

Comment (TimKolberger) [reply website/pages/docs/overlay/modal.mdx:302]: Lets omit the additional "or" ✏️

**Review by TimKolberger — COMMENTED** [review]
There is a small typo left - then we are good to go 🚀

File: website/pages/docs/overlay/modal.mdx (hunk 2 above)
Comment (ytakayanagi) [reply website/pages/docs/overlay/modal.mdx]: I must have been drunk. Fix is now being pushed.

**Review by TimKolberger — APPROVED** [review]
Thank you for your time @ytakayanagi 💖

**Review by with-heart — APPROVED** [review]

**Pull request merged**